grails-app/controllers/org/pih/warehouse/api/ReplenishmentApiController.groovy

```diff
@@ -144,4 +151,38 @@ class ReplenishmentApiController {
         replenishmentService.deleteReplenishmentItem(params.id)
         render status: 204
     }
+
+    def createPicklist = {
+        OrderItem orderItem = OrderItem.get(params.id)
+        if (!orderItem) {
+            throw new IllegalArgumentException("Can't find order item with given id: ${params.id}")
+        }
+
+//        TODO: OBAM-169: Remove user modified picklist for given orderItem
+//        replenishmentService.removePicklist(orderItem)
```

Comment:
Can you explain the commented out code?

Reply:
Each order item in the replenishment will be auto-picked at first, and then the user will have an option to change the pick or go back to the auto-pick (it is the same as currently existing picking for requisitions). These lines are commented out because the auto picking and picking is not yet implemented and it should be added in the mentioned ticket (or in OBAM-173).